Geometry/HcalCommonData/interface/CaloSimParametersFromDD.h

```diff
@@ -2,24 +2,37 @@
 #define HcalCommonData_CaloSimParametersFromDD_h
 
 #include "DetectorDescription/Core/interface/DDsvalues.h"
+#include "DetectorDescription/Core/interface/DDCompactView.h"
+#include "DetectorDescription/DDCMS/interface/DDCompactView.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <string>
+#include <vector>
 
-class DDCompactView;
 class DDFilteredView;
 class CaloSimulationParameters;
 
+template <typename T>
+void myPrint(std::string value, const std::vector<T>& vec) {
+  edm::LogVerbatim("HCalGeom") << "CaloSimParametersFromDD: " << vec.size() << " entries for " << value << ":";
+  unsigned int i(0);
+  for (const auto& e : vec) {
+    edm::LogVerbatim("HCalGeom") << " (" << i << ") " << e;
+    ++i;
+  }
+}
+
 class CaloSimParametersFromDD {
 public:
-  CaloSimParametersFromDD(bool fromDD4Hep);
+  CaloSimParametersFromDD() {}
```

Comment:
```suggestion
  CaloSimParametersFromDD() = default;
```

Reply:
Can you approve this? I shall make this change in next PR